en/deploy/README.md

```diff
@@ -1,246 +1,304 @@
+    *.pyc
+    __pycache__
+    myvenv
+    db.sqlite3
 
-Go ahead and create `mysite/local_settings.py` file. It should contain your `DATABASE` setup from your `mysite/settings.py` file. Just like that:
+> **Note**: The dot on the beginning of the file name is important!
 
-    import os
-    BASE_DIR = os.path.dirname(os.path.dirname(__file__))
+It's a good idea to use a `git status` command before `git add` or whenever you find yourself unsure of what will be done, to prevent any surprises from happening (e.g. wrong files will be added or commited). The `git status` command returns information about any untracked/modifed/staged files, branch status and much more. The output should be similar to:
 
-    DATABASES = {
-        'default': {
-            'ENGINE': 'django.db.backends.sqlite3',
-            'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
-        }
-    }
+    $ git status
+    On branch master
 
-    DEBUG = True
+    Initial commit
 
-Then just save it! :)
+    Untracked files:
+      (use "git add <file>..." to include in what will be committed)
 
-## mysite/settings.py
+      .gitignore
+      blog/
+      manage.py
+      mysite/
 
-Another thing we need to do is modify our website's `settings.py` file. Open `mysite/settings.py` in your editor and add the following lines at the end of the file:
+    nothing added to commit but untracked files present (use "git add" to track)
 
-    import dj_database_url
-    DATABASES['default'] = dj_database_url.config()
+And finally we save our changes. Go to your console and run these commands:
 
-    SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
+    $ git add -A .
+    $ git commit -m "My Django Girls app, first commit"
+     [...]
+     13 files changed, 200 insertions(+)
+     create mode 100644 .gitignore
+     [...]
+     create mode 100644 mysite/wsgi.py
 
-    ALLOWED_HOSTS = ['*']
 
-    STATIC_ROOT = 'staticfiles'
+## Pushing our code to GitHub
 
-    DEBUG = False
+Go to [GitHub.com](http://www.github.com') and sign up for a new, free user account.  Then, create a new repository, giving it the name "my-first-blog".  Leave the "initialise with a README" tickbox un-checked, leave the .gitignore option blank (we've done that manually) and leave the License as None.
 
-    try:
-        from .local_settings import *
-    except ImportError:
-        pass
+<img src="images/new_github_repo.png" />
 
-It'll do necessary configuration for Heroku and also it'll import all of your local settings if `mysite/local_settings.py` exists.
+> **Note** The name `my-first-blog` is important -- you could choose something else, but it's going to occur lots of times in the instructions below, and you'd have to substitute it each time.  It's probably easier to just stick with the name `my-first-blog`.
 
-Then save the file.
+On the next screen, you'll be shown your repo's clone URL.  Choose the "HTTPS" version, copy it, and we'll paste it into the terminal shortly:
```

Comment:
You should use both or explain repo, you very rarely use repository it is almost always shortened to repo.